frontend/src/metabase/hoc/TitleWithLoadingTime.jsx

```diff
@@ -6,8 +6,8 @@ import title from "metabase/hoc/Title";
 const SECONDS_UNTIL_DISPLAY = 10;
 
 export default startTimePropName => ComposedComponent =>
-  title(({ [startTimePropName]: startTime }) => {
-    if (startTime == null) {
+  title(({ [startTimePropName]: startTime, isRunning }) => {
```

Comment:
As we no longer set `startTime` to null when not running, we need to check for `isRunning` here. This prop is set in both places this function is used - both in QB and dashboards